index.html

```diff
@@ -2,174 +2,267 @@
-            <img class="menu" src="img/menu.svg" alt="Menu-bar">
-        </nav>
-    </header>
-    <section class="main">
-        <h1>Lekan Jimoh</h1>
-        <p>Hello! I am a software developer! I can help you build a product, feature or website. Take a look of my
-            works.
-            <br>If you like what you see and have a project you need coded, don't hesitate and contact me.
+  <header>
+    <nav class="navbar ">
+      <img class="icon" src="images/image_geometry_1.svg" alt="logo" />
+      <button class="nav-btn"></button>
+      <ul class="nav-list">
+        <li><a href="#hello"> Hello</a></li>
+        <li><a href="#projects"> Portfolio</a></li>
+        <li><a href="#about"> About</a></li>
+        <li><a href="#contact">Contact</a></li>
+      </ul>
+    </nav>
+
+    <nav class="open-nav disappear">
+      <div class="drawer-image"></div>
+      <div class="drawer-image-1"></div>
+      <div class="drawer-image-2"></div>
+      <div class="drawer-image-3"></div>
+      <button class="close-drawer-btn"> </button>
+      <ul class="drawer-menu-items">
+        <li><a href="#hello"> Hello</a></li>
+        <li><a href="#projects"> Portfolio</a></li>
+        <li><a href="#about"> About</a></li>
+        <li><a href="#contact">Contact</a></li>
+      </ul>
+    </nav>
+  </header>
+  <main>
+    <section class="intro-section" id="hello">
+      <article class="white-bg">
+        <h1 class="heading">Lekan Jimoh</h1>
+        <p class="para">
+          Hello! I am a software developer! I can help you build a product,
+          feature or website. Taka a look of my works. If you like what you
+          see and have a project you need coded, don't hesitate and contact
+          me.
+        </p>
+        <button class="buttons-primary-dark" type="button">
+          Start collaboration
+        </button>
+      </article>
+      <img class="page-brk-dp" src="images/image-geometry_1.svg" alt="page break" />
+    </section>
+    <img class="page-brk" src="images/image_geometry_2.svg" alt="page break" />
+    <section class="works" id="projects">
+      <img class="wi wi1" src="images/image_dots-smaller.svg" alt="#" />
+      <img class="wi wi2" src="images/image-geometry_2.svg" alt="#" />
+      <img class="wi wi3" src="images/Group1.svg" alt="#" />
+      <img class="wi wi4" src="images/Group2.svg" alt="#" />
+      <img class="wi wi5" src="images/image-geometry_5.svg" alt="#" />
+      <img class="wi wi6" src="images/image-geometry_4.svg" alt="#" />
+      <h2 class="heading heading-works">Projects</h2>
+      <article class="dark-bg project1">
+        <img src="images/Rectangle21.png" alt="project 1 preview image" />
+        <div class="project">
+          <h3 class="heading2">Project name goes here</h3>
+          <ul class="lang-container">
+            <li class="languages">HTML/CSS</li>
+            <li class="languages">Ruby on Rails</li>
+            <li class="languages">JavaScript</li>
+          </ul>
```

Comment:
- [ ] Please, consider that when screen size switches from mobile to desktop, the projects section lost the single project information. You can see it below:

> You version
<img width="649" alt="Screen Shot 2022-02-17 at 23 11 16" src="https://user-images.githubusercontent.com/49064106/154554790-4818cec1-e915-4034-af83-6b92ef4d1d6e.png">

> Figma design
<img width="394" alt="Screen Shot 2022-02-17 at 23 07 40" src="https://user-images.githubusercontent.com/49064106/154554874-915471fc-3fde-4c16-8d43-be5cd53dcaeb.png">

![2022-02-17 23 22 29](https://user-images.githubusercontent.com/49064106/154555275-7ca65b1a-cb45-448e-b7a2-9b6a7e8853c8.gif)

Reply:
@elmar8287. I chose to not display the projects section because I didn't know how I would display just one instead of the whole six projects sections. I can only do this by using animation, which isn't required at this stage. If you can suggest a way to do this without animations, It'll be a great help.